example/docker-compose.yaml

```diff
@@ -0,0 +1,49 @@
+version: "3.9"
+services:
+  server:
+    build:
+      context: ./server
+      dockerfile: ../Dockerfile
+    environment:
+      - OTEL_SERVICE_NAME=server
+      - OTEL_RESOURCE_ATTRIBUTES
+    network_mode: host
+    depends_on:
+      - otel-collector
+
+  client:
+    build:
+      context: ./client
+      dockerfile: ../Dockerfile
+    environment:
+      - OTEL_SERVICE_NAME=client
+      - OTEL_RESOURCE_ATTRIBUTES
+    network_mode: host
+    depends_on:
+      - otel-collector
+
+  otel-collector:
+    image: otel/opentelemetry-collector-contrib:0.47.0
```

Comment:
It would be better to use the Splunk distro but I did not have time to try it.

I think this can be done later.

Reply:
It cannot be done

Splunk Otel Collector does not support a Jaeger exporter which I found extremely handy when I am exploratory and smoke testing

```sh
otel-collector_1  | 2022/04/08 11:29:58 main.go:132: application run finished with error: failed to get config: cannot unmarshal the configuration: unknown exporters type "jaeger" for "jaeger" (valid values: [file kafka logging signalfx splunk_hec httpsink otlp otlphttp sapm])
```